stdpopsim/dfe.py

```diff
@@ -39,87 +38,94 @@ class MutationType(object):
     :vartype convert_to_substitution: bool
     """
 
-    dominance_coeff = attr.ib(default=0.5, type=float)
+    dominance_coeff = attr.ib(default=0.5, type=[float, int])
```

Comment:
I think this is why the docs aren't building (when you build, do you at least get a warning?). The error says `error while formatting arguments for stdpopsim.MutationType: unhashable type: 'list'`; perhaps you should use a tuple instead?

OH WAIT: I think that dominance coefficient should be a float - why int? I think attrs does not do anything with this anyhow? (From the docs for `attr.ib( )`:
![Screenshot from 2022-07-21 22-03-41](https://user-images.githubusercontent.com/1046249/180366649-8e68f202-7e1b-4706-bb79-53494eff9577.png)

Reply:
I didn't get any warning about `error while formatting arguments for stdpopsim.MutationType: unhashable type: 'list'` in my computer, only got `RuntimeWarning: overflow encountered in power`.